src/lib/components/UI/PrimaryButton.svelte

```diff
@@ -8,20 +8,7 @@
 		{disabled}
 		on:click
 		type="button"
-		class:opacity-60={disabled}
-		class={`${
-			buttonSize === 'xs'
-				? 'px-2.5 py-1.5 text-xs font-medium'
-				: buttonSize === 'sm'
-				? 'px-3 py-2 text-sm leading-4 font-medium'
-				: buttonSize === 'md'
-				? 'px-4 py-2 text-sm font-medium'
-				: buttonSize === 'lg'
-				? 'px-4 py-2 text-base font-medium'
-				: buttonSize === 'xl'
-				? 'px-6 py-3 text-base font-medium'
-				: ''
-		}inline-flex items-center border border-transparent rounded-md shadow-sm text-white bg-cyan-600 hover:bg-cyan-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-cyan-500`}
+		class="w-40 px-9 py-3 bg-green-500 rounded-3xl text-sm font-semibold text-white cursor-pointer hover:opacity-80"
```

Comment:
We should keep conditionally rendering button like previous version. Please revert to older one and add styling base on the case we need.

Reply:
should be like this:
```
class="{`${
			buttonSize === 'xs'
				? 'px-2.5 py-1.5 text-xs font-medium'
				: buttonSize === 'sm'
				? 'px-3 py-2 text-sm leading-4 font-medium'
				: buttonSize === 'md'
				? 'px-4 py-2 text-sm font-medium'
				: buttonSize === 'lg'
				? 'px-4 py-2 text-base font-medium'
				: buttonSize === 'xl'
				? 'px-6 py-3 text-base font-medium'
				: ''
		} w-40 px-9 py-3 bg-green-500 rounded-3xl text-sm font-semibold text-white cursor-pointer hover:opacity-80"
		```